lua/genghis/init.lua

```diff
@@ -160,19 +160,50 @@ function M.chmodx()
 	cmd.edit()
 end
 
----@param opts? {trashLocation: string}
+---@param opts? {trashCmd: string, trashLocation: string}
 function M.trashFile(opts)
 	cmd.update { bang = true }
 	local home = os.getenv("HOME")
 	local oldFilePath = vim.api.nvim_buf_get_name(0)
 	local oldName = vim.fs.basename(oldFilePath)
 
+	local trashCmd
+	if opts ~= nil and opts.trashCmd ~= nil then
+		trashCmd = opts.trashCmd
+	else
+		if fn.has("linux") == 1 then
+			trashCmd = "gio trash"
+		elseif fn.has("windows") == 1 then
+			trashCmd = "trash"
```

Comment:
is that CLI pre-installed on Windows? If not, I think it'd make more sense to use "move to trash location" as default for Windows as well

Reply:
No but I think moving to a non-standard folder would feel annoying for me at least. I can just change that if you're sure about this?

Reply:
Yeah, iirc, researching the default folder for windows was actually still a todo… 

tbh, I don't even know which is the default folder, but if you know one, it'd make sense to use that over a CLI that is not even available on by default.

Reply:
I believe the case for *Windows* is even more complicated than that of *Linux*. Moving to the recycle bin is not the same as just move the file to a directory.

Reply:
Actually I discovered *trash-cli* doesn't work with *Windows*, so *Windows* users will need to find their own tool. From googling there are quite a few.

Reply:
yeah, I'd still prefer moving to the recycle bin as default behavior, this plugin is intended to work without dependencies. Optionally using a trash cli if the user wants easier restoring of files is fine.

Reply:
In *Windows* you cannot move to the recycle bin at all without copying what these trash utilities do.

The previous behavior just moves files to some folder. Note again it was bugged as well. Doing this twice with files with the same name overwrites them.

I'm very uncomfortable with this but if you insist. I'll definitely have to note this down in the documentation.

Reply:
It does work like that on Windows? weird.

Yeah, I haven't touched a windows device since more than a decade, I just assumed it works like on macOS in this regard. Here, moving a file to the trash works fine, and if the file already exists there, the movement simply fails.

So I am understanding it right, there is *no* way to move a file to the trash on Windows programmatically without a CLI?

Reply:
> without a CLI?

Without copying what they do.

Reply:
Oh boy, I wasn't aware that the situation on Windows was *so bad*, I couldn't even imagine it 😆 

Okay, my bad. In that case, yeah, keep the movement via trash command the default on Windows, too.

Maybe add a sentence in the docs that windows is the only OS that has a required dependency for trashing a file.